source/seismogram/index.rst

```diff
@@ -0,0 +1,9 @@
+地震图
+======
+
+地震图 (seismogram) 是指地震仪记录到的带有记时信息的地面质点运动（位移、速度、加速度、转动等），
+也称为地震波形 (seismic waveform)。其一般有三个正交分量，如南北、东西和垂直分量，
+因此地震图的本质是三维时间序列。
+
+地震图与元数据 (metadata) 共同构成了完备的地震学观测数据。元数据包含与地震图相关的重要信息，
+如台站位置、仪器响应、震源位置、震源机制等。
```

Comment:
I don't think earthquake location and earthquake focal mechanisms belong to metadata.

Reply:
How do you define metadata?

Reply:
metadata is the information of stations or instruments, not earthquakes.

A seismogram can be full of noise without any earthquake information

Reply:
Yes, I agree. metadata is the "data" that are not the real data, while event information and seismic structures are the results we invert from data along with metadata.

Reply:
Yes